src/components/SchedulePage/Event.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles(theme => ({
+	name: {
+		fontFamily: theme.typography.fontFamily,
+		color: theme.palette.secondary.main,
+		fontWeight: 'bold',
+		textTransform: 'uppercase'
+	},
+	location: {
+		fontFamily: theme.typography.fontFamily,
+		fontWeight: theme.typography.fontWeightMedium
+	},
+	description: {
+		fontFamily: theme.typography.fontFamily
+	},
+	container: {
+		paddingBottom: 12
+	}
+}));
+
+function Event({ name, location, description }) {
+	const classes = useStyles();
+	return (
+		<Container maxWidth="md">
```

Comment:
See my comment in `Timeslot.js`. I don't think the `Container` should be needed.